app/main.py

```diff
@@ -21,38 +21,24 @@ def __iadd__(self, other):
         return self
 
     def __mul__(self, other):
-        if isinstance(other, Distance):
-            return Distance(self.km * other.km)
         return Distance(self.km * other)
 
     def __truediv__(self, other):
-        if isinstance(other, Distance):
-            return Distance(round((self.km / other.km), 2))
         return Distance(round((self.km / other), 2))
 
     def __lt__(self, other):
-        if isinstance(other, Distance):
-            return self.km < other.km
         return self.km < other
 
     def __gt__(self, other):
-        if isinstance(other, Distance):
-            return self.km > other.km
         return self.km > other
 
     def __eq__(self, other):
-        if isinstance(other, Distance):
-            return self.km == other.km
         return self.km == other
 
     def __le__(self, other):
-        if isinstance(other, Distance):
-            return self.km <= other.km
         return self.km <= other
 
     def __ge__(self, other):
-        if isinstance(other, Distance):
-            return self.km >= other.km
         return self.km >= other
```

Comment:
and here you should use the previously created magic methods. Look: `<=` is the same as `not >`

If it's hard to understand tag me on your slack channel or visit qna)